modules/MouseModels.py

```diff
@@ -1,6 +1,5 @@
 import requests
 import collections
-from tqdm import tqdm
```

Comment:
@tskir , what are the plans: are we rolling out the `PhenoDigm.py` parser into production for this release? If yes, we can remove this file. If no, we can keep it for one more release.

Reply:
MouseModels.py has been removed.